packages/edit-site/src/components/create-pattern-modal/index.js

```diff
@@ -100,13 +98,11 @@ export default function CreatePatternModal( {
 						__nextHasNoMarginBottom
 					/>
 					<ToggleControl
-						label={ __( 'Synced' ) }
+						label={ __( 'Keep all pattern instances in sync' ) }
 						onChange={ onSyncChange }
-						help={
-							syncType === SYNC_TYPES.full
-								? __( 'Content is synced' )
-								: __( 'Content is not synced' )
-						}
+						help={ __(
+							'Editing the original pattern will also update anywhere the pattern is used.'
+						) }
```

Comment:
Should we update this help text based on whether the toggle is on or not?

At the moment regardless of the state of the toggle switch it seems to indicate that editing the pattern will update anywhere the pattern is used.

Reply:
This has been discussed and the decision was that help text should just describe the enabled state to be consistent with the label.